www/kadm08/sp/newReservation.php

```diff
@@ -0,0 +1,96 @@
+<?php
+session_start();
+
+
+require_once __DIR__ . '/lib/ReservationDB.php';
+require_once __DIR__ . '/lib/UserDB.php';
+require __DIR__ . '/userRequired.php';
+
+$success = false;
+
+$errorMessages = [];
+
+$reservationDB = new ReservationDB();
+$userDB = new UserDB();
+
+$available_clients = $userDB->fetchAllClients(); 
+
+
+if (!empty($_POST)) {
+    if (empty($_POST['start']) || empty($_POST['end'])) {
+        array_push($errorMessages,   'You have to select starting and ending dates.');
+    }
+
+    $available_workplaces = $reservationDB->getAvailableWorkplace(htmlspecialchars($_POST['end']), htmlspecialchars($_POST['start']));
+
+    if ($_POST['start'] > $_POST['end']) {
+        array_push($errorMessages,   'Reservation end cannot be before reservation start.');
```

Comment:
jeste chybi kontrola datumu pro pripad ze uzivatel zvoli jiz probehle

Reply:
Kontrola pridana, mela jsem v planu ji tam nakonec pridat, protoze v prubehu vyvoje se mi hodilo mit tam moznost pridavat rezervace "do minulosti", jen jsem na to nakonec zapomnela.